#3636 Default values removed from regulation .mo, added to .par and examples .mo

ErwanGuichard49 opened the pull request:
closes [#3636](https://github.com/dynawo/dynawo/issues/3636)

joyelfeghali reviewed (changes requested):
Could you add an updateXML file to add these parameters to the .par?

ErwanGuichard49 commented:
> Could you add an updateXML file to add these parameters to the .par?

I have added one such file.

joyelfeghali commented:
> > Could you add an updateXML file to add these parameters to the .par?
> 
> I have added one such file.

Thanks, could you just rename it 1.8.0.1 since it will be merged to v.1.8.

Also could you squash the commit? Thanks

ErwanGuichard49 commented:
> > > Could you add an updateXML file to add these parameters to the .par?
> > 
> > 
> > I have added one such file.
> 
> Thanks, could you just rename it 1.8.0.1 since it will be merged to v.1.8.
> 
> Also could you squash the commit? Thanks

I have just renamed the .py file and squashed the commits.
